TESTING.md

```diff
@@ -0,0 +1,111 @@
+# Testing
+
+The test suite uses [Docker](https://www.docker.com/community-edition) to spin up the following databases when running `npm test`:
+
+| Database      | Service Name |
+|---------------|--------------|
+| PostgreSQL    | `postgres` |
+| MySQL         | `mysql` |
+| MongoDB       | `mongodb` |
+| Elasticsearch | `elasticsearch` |
+| Cassandra     | `cassandra` |
+| SQL Server    | `mssql` |
+| Redis         | `redis` |
```

Comment:
Do you think it's important to include this list here? It's another place we need to remember to keep in-sync with reality, as we add new instrumentation.

Reply:
Good point. We could just say that you have to choose one or more of the services you get by running:

```
docker-compose -f test/docker-compose.yml config --services
```

That will generate this output:

```
postgres
mongodb
mssql
mysql
redis
elasticsearch
cassandra
node_tests
```

Reply:
Sounds good to me. 👍

Reply:
Fixed